src/Compiler/CodeGen/IlxGen.fs

```diff
@@ -11631,11 +11837,12 @@ let CodegenAssembly cenv eenv mgbuf implFiles =
 //-------------------------------------------------------------------------
 
 let GetEmptyIlxGenEnv (g: TcGlobals) ccu =
-    let thisCompLoc = CompLocForCcu ccu
-
     {
         tyenv = TypeReprEnv.Empty
-        cloc = thisCompLoc
+        cloc = CompLocForCcu ccu
+        initClassCompLoc = None
+        initFieldName = CompilerGeneratedName "init"
```

Comment:
Will this be a breaking change if the type already had a field called `init` before?

Reply:
No, it uses marker - "@" suffix.